src/viur/core/bones/relational.py

```diff
@@ -265,17 +255,19 @@ def __init__(
         if self.kind is None or self.module is None:
             raise NotImplementedError("Type and Module of RelationalBone must not be None")
 
+        # Referenced keys
+        self.refKeys = {"key"}
```

Comment:
```suggestion
        self.refKeys = {"key", "name"}
```

Reply:
No, see the default refKeys definition.
There might also be cases, where a `name` is not available.